integration/experiment/launch_util.py

```diff
@@ -97,23 +114,29 @@ def launch_run(agent_conf, app_conf, run_id, output_dir, extra_cli_args,
                                                  num_node=num_nodes,
                                                  num_rank=num_ranks,
                                                  cpu_per_rank=cpu_per_rank)
+    start_time = timeit.default_timer()
     launcher.run()
+    end_time = timeit.default_timer()
 
     # Get app-reported figure of merit
     fom = app_conf.parse_fom(log_path)
+    # Get total runtime
+    total_runtime = datetime.timedelta(seconds=end_time-start_time).total_seconds()
     # Append to report
     with open(report_path, 'a') as report:
         report.write('\nFigure of Merit: {}\n'.format(fom))
+        report.write('Total Runtime: {}\n'.format(total_runtime))
```

Comment:
Thank you for adding this.

Reply:
I got tired of forgetting to add it to my scripts.  What might be interesting for later analysis is that this number is longer than the application total runtime in the report because that's the time between MPI_Init and MPI_Finalize.